internal/processors/plugin/config.go

```diff
@@ -0,0 +1,25 @@
+// Copyright Mia srl
```

Comment:
plugin is a little too generic, can you propose another processor name?

Reply:
do you have something in mind?

- dynamicplugin
- customprocessor

?

What about the processor type constant also?